site/source/docs/contributing/developers_guide.rst

```diff
@@ -104,37 +104,54 @@ Bisecting
 If you find a regression, bisection is often the fastest way to figure out what
 went wrong. This is true not just for finding an actual regression in Emscripten
 but also if your project stopped working when you upgrade, and you need to
-investigate if it's an Emscripten regression or something else.
+investigate if it's an Emscripten regression or something else. The rest of
+this section covers bisection on Emscripten itself.
 
-The normal ``git bisect`` workflow can work if you just need to bisect a single
-repository. For example, you can bisect only on the emscripten repo if you are on
-a range that all works with the same versions of LLVM and Binaryen (which was
-mentioned earlier, are two separate codebases that are depended on).
+If you have a large bisection range - for example, that covers more than one
+version of Emscripten - then you probably have changes across multiple repos
+(Emscripten, LLVM, and Binaryen). In that case the easiest and fastest thing
+is to bisect using **emsdk builds**. Each step of the bisection will download
+a "tip of tree" (tot) build from the emsdk. Using this approach you don't need
```

Comment:
I don't think we need to mention tot in this paragraph.  How about

`a build produced by the emscripten releases builders.`

or

`a build based on the history of emscripten releases`
?